agent/config/pbench-agent-default.cfg

```diff
@@ -2,6 +2,10 @@
 version = 002
 pbench_web_server = pbench.example.com
 
+light-tool-set = sar, iostat, mpstat, proc-vmstat, turbostat
```

Comment:
I am not sure how "light" this is really, but `mpstat` and `sar` data overlap with both capturing CPU statistics; `turbostat` data is "Intel" CPUs only, and `proc-vmstat` is "a lot" of VM statistics.

Perhaps we can consider something like:

```
[pbench/tools]
legacy-tool-set = sar, iostat, mpstat, pidstat, proc-vmstat, proc-interrupts, turbostat, perf
light-tool-set = vmstat
medium-tool-set = %(light-tool-set)s, iostat, sar
default-tool-set = %(medium-tool-set)s
heavy-tool-set = %(medium-tool-set)s, mpstat, perf, pidstat, proc-interrupts, proc-vmstat, turbostat
```

The thinking is that `vmstat` gives you CPU, memory, and I/O stats (not networking per sae) for a very "light-weight" data collection, while "medium" gives you a healthy set of CPU (`vmstat`, `sar`), memory (`vmstat`, `sar`), disk (`sar`, `iostat`), networking (`sar`), with "heavy" rounding it out with the rest of the kitchen sink.

But even so, I'd rather round all this out using PCP/Prometheus mappings than tools names, but that is for another PR.